Scripts/RemediationScripts/Remediate-EnableAADOnlyAuthenticationForSQLServers.ps1

```diff
@@ -1,12 +1,12 @@
 <###
 # Overview:
```

Comment:
Getting error while running the BRS, pls help to verify the versions of module and then install accordingly.

Reply:
Thanks Richa, addressed.